### DISCUSSION TITLE
Thank you!

### DISCUSSION THREAD

**Discussion by sv87411** [AT0myks/reolink-fw-archive#26] [General]
I just want to express my thanks to you for maintaining this archive.

It frustrates me tremendously that Reolink only offer the latest firmware for devices on their download site. It means that if you apply a firmware and it has an issue, unless you personally keep a copy of the previous firmware or have the ability to download one from somewhere, you are up the creek!

I recently applied the latest Trackmix PoE firmware that Reolink provide (and also some of the previous versions on your archive) and they completely mess up viewing the Tracking stream. I have had to revert all the way back to the original camera firmware that was on my cameras (July '22), from your archive, which resolves the issue.

So again. THANK YOU! Keep up the excellent work!

**Comment by AT0myks** [comment]
Yeah I don't get why they only provide the latest, I'm sure the support team loves reading emails and posts every day about where to find an older version to fix some issue. I really think they're doing a disservice to themselves. Maybe it wouldn't be as bad if the firmwares were always stable and better than the previous one, but unfortunately that's not the case in (too) many instances.
I appreciate that you took the time to write this, glad I could help!
